lib/cdo/delete_accounts_helper.rb

```diff
@@ -20,19 +20,14 @@ def initialize(solr: nil)
   end
 
   # Deletes all project-backed progress associated with a user.
-  # @param [Integer] user_id The user to delete the project-backed progress of.
-  def delete_project_backed_progress(user_id)
-    user_storage_ids = @pegasus_db[:user_storage_ids]
-    storage_apps = @pegasus_db[:storage_apps]
-    storage_id = user_storage_ids.where(user_id: user_id).first&.[](:id)
-    return unless storage_id
-
-    # Soft-delete all of the user's channels
-    storage_apps.where(storage_id: storage_id).update(
-      state: 'deleted',
-      value: nil,
-      updated_ip: ''
-    )
+  # @param [User] user The user to delete the project-backed progress of.
+  def delete_project_backed_progress(user)
```

Comment:
this file may make sense to move to dashboard/lib since it relies on a dashboard user, although it's possible there's a reason to keep it out of dashboard that I'm not aware of.

Reply:
I've got mixed feelings about this but either way I'd rather make that change in a separate PR.

One one hand, this is very much a cross-cutting concern, with dependencies on dashboard- and pegasus-specific code.

On the other hand, I've already got the tests for this file in dashboard because it's been much easier to test as part of that suite... and things in dashboard (like `User`) already refer to the pegasus db directly.